src/recipes/conditionalMultiline.ts

```diff
@@ -0,0 +1,32 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const conditionalMultilineRecipe = defineRecipe({
+  className: 'multiline',
+  description: 'meant to override other styles when applied',
+  base: {},
+  variants: {
+    apply: {
```

Comment:
For applying or not applying an entire recipe, it would be better to just use `base` styles and put the condition in React:

```ts
cx(multiline ? multilineRecipe() : null, ...)
```

Reply:
Fixed!